diff --git a/simulation/libsimulator b/simulation/libsimulator
index 5ef60542f0..12a83d04f2 160000
--- a/simulation/libsimulator
+++ b/simulation/libsimulator
@@ -1 +1 @@
-Subproject commit 5ef60542f004bbdf142bb4d100ae2b564ec923f7
+Subproject commit 12a83d04f2ffec2dfdf23b0f3b63a148628d939a
diff --git a/src/session_impl.cpp b/src/session_impl.cpp
index f8a7eeecc3..87d27e862b 100644
--- a/src/session_impl.cpp
+++ b/src/session_impl.cpp
@@ -285,17 +285,16 @@ void apply_deprecated_dht_settings(settings_pack& sett, bdecode_node const& s)
 		, std::vector<listen_endpoint_t>& eps)
 	{
 		auto unspecified_begin = std::partition(eps.begin(), eps.end()
-			, [](listen_endpoint_t const& ep) { return !ep.addr.is_unspecified(); });
+			, [](listen_endpoint_t const& ep) { return !(ep.addr.is_v6() && ep.addr.is_unspecified()); });
 		std::vector<listen_endpoint_t> unspecified_eps(unspecified_begin, eps.end());
 		eps.erase(unspecified_begin, eps.end());
 		for (auto const& uep : unspecified_eps)
 		{
-			bool const v4 = uep.addr.is_v4();
 			for (auto const& ipface : ifs)
 			{
 				if (!ipface.preferred)
 					continue;
-				if (ipface.interface_address.is_v4() != v4)
+				if (ipface.interface_address.is_v4())
 					continue;
 				if (!uep.device.empty() && uep.device != ipface.name)
 					continue;
