diff -rupN qBittorrent-up/src/gui/uithememanager.cpp qBittorrent/src/gui/uithememanager.cpp
--- qBittorrent-up/src/gui/uithememanager.cpp
+++ qBittorrent/src/gui/uithememanager.cpp
@@ -307,10 +307,10 @@ void UIThemeManager::applyStyleSheet() const

 void UIThemeManager::onColorSchemeChanged()
 {
-    emit themeChanged();
     // workaround to refresh styled controls once color scheme is changed
     QApplication::setStyle(QApplication::style()->name());
+    emit themeChanged();
 }

 QIcon UIThemeManager::getIcon(const QString &iconId, [[maybe_unused]] const QString &fallback, const ColorMode colorMode) const
