diff --git a/src/gui/mainwindow.cpp b/src/gui/mainwindow.cpp
index 51d380a..d69144e 100644
--- a/src/gui/mainwindow.cpp
+++ b/src/gui/mainwindow.cpp
@@ -264,6 +264,9 @@ MainWindow::MainWindow(QWidget *parent)
 #ifdef Q_OS_MACOS
     // Increase top spacing to avoid tab overlapping
     m_ui->centralWidgetLayout->addSpacing(8);
+
+    // Fix status bar border rendering till the middle of the screen.
+    setStyleSheet("QStatusBar {border-top: 1px solid grey;}");
 #endif
 
     m_ui->centralWidgetLayout->addWidget(m_tabs);
diff --git a/src/gui/statusbar.cpp b/src/gui/statusbar.cpp
index c990cca..7090f9f 100644
--- a/src/gui/statusbar.cpp
+++ b/src/gui/statusbar.cpp
@@ -141,7 +141,7 @@ StatusBar::StatusBar(QWidget *parent)
     layout->addWidget(m_upSpeedLbl);
 
     addPermanentWidget(container);
-    setStyleSheet("QWidget {margin: 0;}");
+    setStyleSheet("QWidget {margin: 1;}");
     container->adjustSize();
     adjustSize();
     // Is DHT enabled
