diff -rupN qBittorrent-up/src/gui/mainwindow.cpp qBittorrent/src/gui/mainwindow.cpp
--- qBittorrent-up/src/gui/mainwindow.cpp
+++ qBittorrent/src/gui/mainwindow.cpp
@@ -1665,7 +1665,7 @@
     {
         const QString msg {tr("A new version is available.") + u"<br/>"
             + tr("Do you want to download %1?").arg(newVersion) + u"<br/><br/>"
-            + u"<a href=\"https://www.qbittorrent.org/news\">%1</a>"_s.arg(tr("Open changelog..."))};
+            + u"<a href=\"https://github.com/vit9696/qBittorrent/releases\">%1</a>"_s.arg(tr("Open changelog..."))};
         auto *msgBox = new QMessageBox {QMessageBox::Question, tr("qBittorrent Update Available"), msg
             , (QMessageBox::Yes | QMessageBox::No), this};
         msgBox->setAttribute(Qt::WA_DeleteOnClose);
