diff -rupN qBittorrent-up/src/gui/properties/propertieswidget.cpp qBittorrent/src/gui/properties/propertieswidget.cpp
--- qBittorrent-up/src/gui/properties/propertieswidget.cpp
+++ qBittorrent/src/gui/properties/propertieswidget.cpp
@@ -76,7 +76,7 @@
     , m_ui(new Ui::PropertiesWidget())
 {
     m_ui->setupUi(this);
-    setAutoFillBackground(true);
+    setAutoFillBackground(false);

     m_state = VISIBLE;
     
