diff -rupN qBittorrent-up/src/base/version.h.in qBittorrent/src/base/version.h.in
--- qBittorrent-up/src/base/version.h.in	2021-11-05 20:26:21.000000000 +0300
+++ qBittorrent/src/base/version.h.in	2021-11-05 20:25:36.000000000 +0300
@@ -29,10 +29,10 @@
 #pragma once

 #define QBT_VERSION_MAJOR 4
-#define QBT_VERSION_MINOR 6
-#define QBT_VERSION_BUGFIX 0
+#define QBT_VERSION_MINOR 5
+#define QBT_VERSION_BUGFIX 3
 #define QBT_VERSION_BUILD 0
-#define QBT_VERSION_STATUS "alpha1"  // Should be empty for stable releases!
+#define QBT_VERSION_STATUS ""  // Should be empty for stable releases!

 #define QBT__STRINGIFY(x) #x
 #define QBT_STRINGIFY(x) QBT__STRINGIFY(x)
--- qBittorrent-up/dist/mac/Info.plist	2021-11-06 00:50:50.000000000 +0300
+++ qBittorrent/dist/mac/Info.plist	2021-11-06 01:03:01.000000000 +0300
@@ -55,7 +55,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.5.0</string>
+	<string>4.5.0</string>
 	<key>CFBundleExecutable</key>
 	<string>${EXECUTABLE_NAME}</string>
 	<key>CFBundleIdentifier</key>
