diff -rupN qBittorrent-up/src/gui/uithememanager.cpp qBittorrent/src/gui/uithememanager.cpp
--- qBittorrent-up/src/gui/uithememanager.cpp
+++ qBittorrent/src/gui/uithememanager.cpp
@@ -53,15 +53,16 @@
 {
     bool isDarkTheme()
     {
+        if (const Qt::ColorScheme scheme = qApp->styleHints()->colorScheme()
+                ; scheme != Qt::ColorScheme::Unknown)
+        {
+            return (scheme == Qt::ColorScheme::Dark);
+        }
+
         const QPalette palette = qApp->palette();
         const QColor &color = palette.color(QPalette::Active, QPalette::Base);
         return (color.lightness() < 127);
     }
-
-    Path resolveThemePath(const Path &themePath)
-    {
-        return (themePath.isAbsolute() ? themePath : (Profile::instance()->location(SpecialFolder::Config) / themePath));
-    }
 }

 UIThemeManager *UIThemeManager::m_instance = nullptr;
