diff -rupN qBittorrent-up/src/gui/transferlistmodel.cpp qBittorrent/src/gui/transferlistmodel.cpp
--- qBittorrent-up/src/gui/transferlistmodel.cpp	2021-11-05 20:26:21.000000000 +0300
+++ qBittorrent/src/gui/transferlistmodel.cpp	2021-11-05 20:25:36.000000000 +0300
@@ -709,31 +709,37 @@ QColor getDefaultColorByState(const BitT
     case BitTorrent::TorrentState::DownloadingMetadata:
     case BitTorrent::TorrentState::ForcedDownloadingMetadata:
         if (!dark)
-            return {34, 139, 34}; // Forest Green
+            return {0, 0, 0};
         else
-            return {50, 205, 50}; // Lime Green
+            return {255, 255, 255};
     case BitTorrent::TorrentState::StalledDownloading:
     case BitTorrent::TorrentState::StalledUploading:
         if (!dark)
-            return {0, 0, 0}; // Black
+            return {0, 0, 0};
         else
-            return {204, 204, 204}; // Gray 80
+            return {255, 255, 255};
     case BitTorrent::TorrentState::Uploading:
     case BitTorrent::TorrentState::ForcedUploading:
         if (!dark)
-            return {65, 105, 225}; // Royal Blue
+            return {0, 0, 0};
         else
-            return {99, 184, 255}; // Steel Blue 1
+            return {255, 255, 255};
     case BitTorrent::TorrentState::PausedDownloading:
-        return {250, 128, 114}; // Salmon
+        if (!dark)
+            return {0, 0, 0};
+        else
+            return {255, 255, 255};
     case BitTorrent::TorrentState::PausedUploading:
         if (!dark)
-            return {0, 0, 139}; // Dark Blue
+            return {0, 0, 0};
         else
-            return {79, 148, 205}; // Steel Blue 3
+            return {255, 255, 255};
     case BitTorrent::TorrentState::Error:
     case BitTorrent::TorrentState::MissingFiles:
-        return {255, 0, 0}; // red
+        if (!dark)
+            return {0, 0, 0};
+        else
+            return {255, 255, 255};
     case BitTorrent::TorrentState::QueuedDownloading:
     case BitTorrent::TorrentState::QueuedUploading:
     case BitTorrent::TorrentState::CheckingDownloading:
@@ -741,11 +747,14 @@ QColor getDefaultColorByState(const BitT
     case BitTorrent::TorrentState::CheckingResumeData:
     case BitTorrent::TorrentState::Moving:
         if (!dark)
-            return {0, 128, 128}; // Teal
+            return {0, 0, 0};
         else
-            return {0, 205, 205}; // Cyan 3
+            return {255, 255, 255};
     case BitTorrent::TorrentState::Unknown:
-        return {255, 0, 0}; // red
+        if (!dark)
+            return {0, 0, 0};
+        else
+            return {255, 255, 255};
     default:
         Q_ASSERT(false);
         return {255, 0, 0}; // red
