diff --git a/CMakeLists.txt b/CMakeLists.txt
index c163c8c0e..881fca18d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,6 +26,7 @@ include(FeatureOptionsSetup)
 feature_option(GUI "Build GUI application" ON)
 feature_option(WEBUI "Enable built-in HTTP server for remote control" ON)
 feature_option(QT6 "Build with Qt 6 instead of Qt 5" OFF)
+feature_option(DBUS "Use DBUS" ON)
 feature_option(STACKTRACE "Enable stacktrace support" ON)
 feature_option(TESTING "Build internal testing suite" OFF)
 feature_option(VERBOSE_CONFIGURE "Show information about PACKAGES_FOUND and PACKAGES_NOT_FOUND in the configure output (only useful for debugging the CMake build scripts)" OFF)
