diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,6 +21,7 @@
 include(FeatureOptionsSetup)

 # features, list is loosely sorted by user's interests
+feature_option(DBUS "Use DBUS" ON)
 feature_option(GUI "Build GUI application" ON)
 feature_option(WEBUI "Enable built-in HTTP server for remote control" ON)
 feature_option(PLUGINS "Enable built-in Lua plugins engine" ON)
